helm-service/controller/configuration_changer.go

```diff
@@ -410,6 +413,23 @@ func (c *ConfigurationChanger) changeCanary(e *keptnevents.ConfigurationChangeEv
 	return nil
 }
 
+func (c *ConfigurationChanger) scaleDownCanaryDeployment(e *keptnevents.ConfigurationChangeEventData) error {
```

Comment:
As this is new code, please add a short docstring about what this function does, e.g.:
`// get the deployment specified in the event and update it with Replicas: 0`